zh_CN.json

```diff
@@ -0,0 +1,167 @@
+{
+  "__LANGUAGE_INFO": {
+    "name": "中文",
+    "author": "（作者）",
```

Comment:
Please write the `auhor` not in chinese lanuage.
This entrie is for users to contact you for support.